distribution/linux/flatpak/io.github.openenroth.openenroth.yml

```diff
@@ -0,0 +1,85 @@
+app-id: io.github.openenroth.openenroth
+runtime: org.freedesktop.Platform
+runtime-version: &runtime-version '24.08'
+sdk: org.freedesktop.Sdk
+command: io.github.openenroth.openenroth.sh
+
+finish-args:
+  # hardware 3D and controller access
+  - --device=all
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Audio
+  - --socket=pulseaudio
+
+cleanup-commands:
+  - mkdir -p ${FLATPAK_DEST}/lib/ffmpeg
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm18
+
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    add-ld-path: .
+    version: *runtime-version
+    no-autodownload: false
+    autodelete: false
+
+modules:
+  - name: libdwarf
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/davea42/libdwarf-code/
+        tag: v0.11.0
+        commit: 285d9d34f3e9f56cc1c487d0055f6dc54a9c54a1
+        x-checker-data:
+          type: git
+          tag-pattern: ^v\d+(\.\d+)+.*$
+
+  - name: openenroth
+    buildsystem: cmake-ninja
+    builddir: true
+    build-options:
+      append-path: /usr/lib/sdk/llvm18/bin
+      prepend-ld-library-path: /usr/lib/sdk/llvm18/lib
+    cleanup:
+      - /share/cmake
+      - /lib/cmake
+      - /share/man
+      - /share/gch
+      - '*.a'
+      - /include
+    config-opts:
+      - -DOE_BUILD_TOOLS=OFF
+      - -DOE_BUILD_TESTS=OFF
+      - -DOE_CHECK_STYLE=OFF
+      - -DOE_CHECK_LUA_STYLE=OFF
+      - -DOE_USE_PREBUILT_DEPENDENCIES=OFF
+      - -DCMAKE_BUILD_TYPE=RelWithDbgInfo
+      - -DCMAKE_LINKER=lld
+    sources:
+      - type: git
+        url: https://github.com/OpenEnroth/OpenEnroth.git
+        branch: master
+      # Once release tags are created in the OpenEnroth repo and OE is submitted to Flathub, delete the above and use this instead
+      # - type: git
+      #   url: https://github.com/OpenEnroth/OpenEnroth.git
+      #   tag: v0.1
+      #   x-checker-data:
+      #     type: git
+      #     tag-pattern: ^v\d+(\.\d+)+.*$
+    post-install:
+      - install -Dm755 src/Bin/OpenEnroth/OpenEnroth /app/bin/OpenEnroth
+      # launcher script
+      - install -Dm755 ../distribution/linux/flatpak/${FLATPAK_ID}.sh /app/bin/${FLATPAK_ID}.sh
```

Comment:
I'm getting a bit lost on dir structure here, why the `..`? Given that we don't have `..` before `src/Bin/OpenEnroth/OpenEnroth` above.